packages/ui-components/src/components/NatPurseSelector.jsx

```diff
@@ -9,9 +9,9 @@ import { stringifyValue } from '../display/index.js';
  * components represent initial state as null/undefined.
  *
  * @param {Array<PursesJSONState> | null} purses Unfiltered purses. This may be null to simplify use in UIs.
- * @param {Brand} [optBrand] - optional brand to filter for
+ * @param {import('@agoric/ertp/src/types-module').Brand} [optBrand] - optional brand to filter for
```

Comment:
This is exactly what I want to avoid. We should have 
```suggestion
 * @param {import('@agoric/ertp').Brand} [optBrand] - optional brand to filter for
```

Reply:
I agree that's the goal. I just didn't want to have to remake all the ambients. But I accept that as a requirement for explicit package exports.